backpack/extensions/secondorder/diag_ggn/__init__.py

```diff
@@ -132,3 +132,59 @@ def __init__(self, mc_samples=1):
 
     def get_num_mc_samples(self):
         return self._mc_samples
+
+
+class BatchDiagGGN(BackpropExtension):
+    VALID_LOSS_HESSIAN_STRATEGIES = [
+        LossHessianStrategy.EXACT,
+        LossHessianStrategy.SAMPLING,
+    ]
+
+    def __init__(self, loss_hessian_strategy, savefield):
+        self.loss_hessian_strategy = loss_hessian_strategy
```

Comment:
Could you introduce a check for `loss_hessian_strategy` similar to [here](https://github.com/f-dangel/backpack/blob/development/backpack/extensions/secondorder/diag_ggn/__init__.py#L58-L64)?

Reply:
hi, in a previous comment you asked me to remove this because it makes more sense not to have default values.

Reply:
That's true. But I am referring to reintroducing the check, rather than the default value

Reply:
Added